docs/kafka/metrics-monitoring-kafka/README.adoc

```diff
@@ -108,6 +108,12 @@ Topic metrics::
 * `kafka_server_brokertopicmetrics_messages_in_total`: Number of messages per second received by one or more topics in the instance.
 
 * `kafka_topic:kafka_log_log_size:sum`: Log size of each topic and replica, in bytes, across all brokers in the cluster.
```

Comment:
@pawelpaszki So sorry for the delay! I somehow missed the ping :(  You and I already worked through some questions in chat, so my only suggestion is to add periods to the end of each one that you added for consistency, and suggest to move the log size topic metric to the end, to keep the related metrics together.